src/jubilation-horizontal-bar/index.jsx

```diff
@@ -0,0 +1,124 @@
+// @flow
+import React from 'react';
+import uuid from 'uuid';
+import getDomainMap from '../helpers/scatter';
```

Comment:
Probably out of scope, but looking at the `getDomainMap` function, I have the same response as elsewhere- can we replace the oneliner `reduce` calls with a single `Math.max`/`Math.min` call on the mapped array?